packages/react-native-sdk/docusaurus/docs/reactnative/03-core/02-joining-creating-calls.mdx

```diff
@@ -14,7 +14,7 @@ This guide shows how to create, join, leave, and end calls and ring calls.
 
 You can create a call by specifying its `callType` and `callId`:
 
-The [Call Type](../../guides/configuring-call-types) controls which features are enabled, and sets up permissions.
```

Comment:
`configuring-call-types` does not exist as a file, I assumed this was referring to `call-types` in general.

Reply:
Ah, I just noticed the `id` at the top of every `mdx` file, I assume that's what links these ?

Reply:
this is a shared page, it lives in our docusaurus repo: https://github.com/GetStream/stream-chat-docusaurus-cli/blob/production/shared/video/_call-types.mdx

Feel free to take the link from the React docs.